### PR TITLE
Plugins: Disable install button during automated transfer

### PR DESCRIPTION
**Pull request by roundhill**
We received a few reports that the Install button was still enabled during an automated transfer. This PR connects up to the `isTransferring` state and disables the button if a Transfer is occurring.

**To test**
* On a wp.com site with a business plan, visit `/plugins/browse` and select a plugin.
* Set the state in the browser console: `dispatch( { siteId: yoursiteid, type: 'AUTOMATED_TRANSFER_STATUS_SET', status: 'start' } )`
* You should see the transfer dialog, and the Install button disabled.
* Reset the transfer state with another console command: `dispatch( { siteId: yoursiteid, type: 'AUTOMATED_TRANSFER_STATUS_SET', status: '' } )`
* The install button should be enabled again.

Screenshot:
screen shot 2017-03-07 at 11 11 42 am

### CODE REVIEW

File: client/my-sites/plugins/plugin-meta/index.jsx [hunk 1]
```diff
@@ -165,10 +168,12 @@ export default React.createClass( {
 			return <PluginInstallButton { ...this.props } />;
 		}
 
+		const { isTransferring } = this.props;
+
 		if ( this.props.selectedSite && ! this.props.selectedSite.jetpack ) {
 			return (
 				<WpcomPluginInstallButton
-					disabled={ ! this.hasBusinessPlan() }
+					disabled={ ! this.hasBusinessPlan() || isTransferring }
```
Comment (dmsnell) [inline client/my-sites/plugins/plugin-meta/index.jsx]: it shouldn't matter here but just be aware that the value of `isTransferring` here could also be `null`

File: client/my-sites/plugins/plugin-meta/index.jsx [hunk 2]
```diff
@@ -37,9 +38,11 @@ import {
...
 } from 'lib/products-values';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { isAutomatedTransferActive } from 'state/selectors';
 import QueryEligibility from 'components/data/query-atat-eligibility';
 
-export default React.createClass( {
+const PluginMeta = React.createClass( {
 	OUT_OF_DATE_YEARS: 2,
 
 	displayName: 'PluginMeta',
```
Comment (dmsnell) [inline client/my-sites/plugins/plugin-meta/index.jsx]: Giving a name here to the React class means `displayName` is redundant. Might as well remove it

Comment (roundhill) [inline client/my-sites/plugins/plugin-meta/index.jsx]: Right! Removed in 164e1b6

**Comment by dmsnell** [comment]
:shipit:

**Pull request merged**